manifests/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+commonLabels:
+  app: rhoai-edge-pipelines
+  app.kubernetes.io/part-of: rhoai-edge-pipelines
+resources:
+- tasks/
+- aiedge-e2e.pipeline.yaml
+- gitops-update-pipeline.yaml
```

Comment:
For organzational purposes, I would recommending moving the pipelines to a `manifests/pipelines` folder that can deploy the pipelines and required tasks.